src/ss_fact.jl

```diff
@@ -76,8 +131,8 @@ function sspdmp_inner!(Ξ, G, G2, ∇ϕ, t, x, θ, Q, c, b, t_old, f, θf, (acc,
                     acc = num = 0
                     adapt!(c, i, factor)
                 end
-                t, x, θ = smove_forward!(G, i, t, x, θ, t′, F) # neighbours
-                t, x, θ = smove_forward!(G2, i, t, x, θ, t′, F) # neighbours of neightbours \ neighbours
+                # already done above t, x, θ = smove_forward!(G, i, t, x, θ, t′, F) # neighbours
+                t, x, θ = ssmove_forward!(G2, i, t, x, θ, t′, F) # neighbours of neightbours \ neighbours
                 θ = reflect!(i, x, θ, F)
```

Comment:
```suggestion
                θ = reflect!(i, ∇ϕi, x, θ, F)
```
check if `∇ϕi` is still the gradient in `x`, `i`